src/components/App.tsx

```diff
@@ -3,22 +3,43 @@ import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom'
 import 'normalize.css'
 import '@blueprintjs/core/lib/css/blueprint.css'
 import '@blueprintjs/icons/lib/css/blueprint-icons.css'
-import Places from './Places'
-import Payments from './Payments'
+import PlacesPage from './PlacesPage'
+import PaymentsPage from './PaymentsPage'
+import MembersPage from './MembersPage'
+import styled from 'styled-components'
+
+const LinksWrapper = styled.div`
```

Comment:
Recommendation: try to avoid words "wrapper", "container", etc...
This is the navigation section. Or the menu.
Can be SiteMenu as well.